config/repo-values.yaml

```diff
@@ -92,3 +92,5 @@ sync:
       url: https://helm.mattermost.com
     - name: emberstack
       url: https://emberstack.github.io/helm-charts
+    - name: gomods
```

Comment:
@rimusz can you make this and the below `athens`?

Reply:
done